game-core/src/test/java/games/strategy/triplea/ai/pro/util/MoveBatcherTest.java

```diff
@@ -67,27 +71,28 @@ public void testMoveUnitsWithMultipleTransports() {
     //   Move to load all the land units onto transports.
     //   Move transporting all the units.
     //   Move unloading the land units from the transports.
-    assertEquals(3, moves.size());
+    assertThat(moves, hasSize(3));
 
     // Check move to load all the land units onto transports.
-    assertEquals(
+    var expected =
         new MoveDescription(
             List.of(inf1, tank1, tank2, inf2),
             brazilToSz18,
-            Map.of(inf1, transport1, tank1, transport1, inf2, transport2, tank2, transport2)),
-        moves.get(0));
+            Map.of(inf1, transport1, tank1, transport1, inf2, transport2, tank2, transport2));
+    assertThat(moves.get(0), is(expected));
```

Comment:
side-note: This is not a bad place where a string comment would help spell out why we expect this assertion to hold.